addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -45,6 +45,17 @@ $o-wblog-loader-size: 50px;
 // Website Blog
 //------------------------------------------------------------------------------
 .website_blog {
+    .o_website_blog_nav_container {
+        // Overrides the value from portal, useless in >=17
+        &::before, &::after {
+            display: none !important;
+        }
+
+        .dropdown-toggle span {
+            max-width: 20vw;
```

Comment:
Nope, we're not gonna do that 🙅🏻‍♂️😅 
Two issues:
1. Very specific rule but generic selector.
2. Unnecessary "arbitrary value" (why 20 and nor 19?) 

This is a layout issue, we have to adjust the layout instead.

Hint: set the dropdown to  `min-width: 0`; 
ref.https://css-tricks.com/flexbox-truncated-text/#aa-the-solution-is-min-width-0-on-the-flex-child

Reply:
🧙‍♂️ Nice trick, thanks!